src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift

```diff
@@ -8,28 +8,41 @@ import HealthCertificateToolkit
 extension Name {
 
 	var fullName: String {
-		var givenName = self.givenName
-		var familyName = self.familyName
+		return [resolvedGivenName, resolvedFamilyName].formatted()
+	}
+	
+	var reversedFullName: String {
+		return [resolvedFamilyName, resolvedGivenName].formatted()
```

Comment:
I think you need an additional comma in the separator in this case

Reply:
Not according to the ticket

Reply:
I see the comma in the ticket:

> **Expected result:**
> Last Name, First Name
> **Actual result:**
> First Name Last Name

Reply:
I don't see example screenshots, only the photo of the devices with the wrong formatting.

Regardless, I found that the tech spec has been updated with the new desired formatting: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/7e46800f8d9b4c0f0efb09d624960a6476b2f534/docs/spec/dgc-overview-client.md#determining-the-formatted-name-of-the-person-from-a-dgc

Reply:
Done